couchbase_test.go

```diff
@@ -16,10 +16,12 @@ import (
 	dbtesting "github.com/hashicorp/vault/sdk/database/dbplugin/v5/testing"
 	"github.com/ory/dockertest/v3"
 	dc "github.com/ory/dockertest/v3/docker"
+	"github.com/rogpeppe/go-internal/semver"
```

Comment:
Maybe use [hashicorp/go-version](https://github.com/hashicorp/go-version) instead?

Reply:
Oh, I had meant to use the golang.org/x one. Not sure why this got pulled in. Thanks for the catch.